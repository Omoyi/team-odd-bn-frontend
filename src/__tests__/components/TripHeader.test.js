import React from 'react';
import { shallow } from 'enzyme';
import TripHeader from '../../components/trips/TripHeader';

describe('Trip Header Component test with Enzyme', () => {
  test('renders without crashing', () => {
    expect(
      shallow(
        <TripHeader />,
      ),
    ).toMatchSnapshot();
  });
  test('renders without crashing', () => {
    expect(shallow(<TripHeader />)).toMatchSnapshot();
  });
});
