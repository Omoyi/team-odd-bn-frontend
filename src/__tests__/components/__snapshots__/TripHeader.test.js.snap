// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trip Header Component test with Enzyme renders without crashing 1`] = `ShallowWrapper {}`;

exports[`Trip Header Component test with Enzyme renders without crashing 2`] = `ShallowWrapper {}`;
